### DISCUSSION TITLE
Problem saving to Secondary 2TB SSD drive

### DISCUSSION THREAD

**Discussion by 1artist** [photoprism/photoprism#2779] [Q&A]
I installed PhotoPrism as an automated install process using the CasaOS app install feature.  My CasaOS runs on top of the Raspberry Pi 64bit OS.  Additionally, I am using a Raspberry Pi 4B 8GB single board computer with a 64GB Micro SD card as a primary boot drive and a secondary 2TB SSD drive to store my data in. 

Here is where I am confused and would appreciate any advice. I changed the PhotoPrism docker-compose volume settings to what is shown below for the purpose of saving my photos onto my 2tb secondary drive and not the 64GB Micro SD boot drive:

   ```
                                        HOST                                         CONTAINER
/mnt/Usb2tb/storage:/DATA/AppData/PhotoPrism/photoprism/storage     |           /photoprism/storage
/mnt/Usb2tb/originals:/DATA/Gallery                                 |           /photoprism/originals

```

I then uploaded into a PhotoPrism test album a single test JPG file name:  "artist.jpg".  No error message appeared informing me that my revised container path to my 2TB drive was invalid.  So I assumed my test "artist.jpg" file successfully got uploaded to the new 2TB SSD drive location. 

However, when I SSH using Putty to my 2tb drive and start fishing around for the newly uploaded "artist.jpg" test inside the "storage:" folder and the "originals:" folder my jpg test file it is nowhere to be found.  

I am a novice with all of this docker container stuff.  Can anybody in the community please tell me how PhotoPrism stores the data onto my 2TB SSD drive?  Since CasaOS automatically installed PhotoPrism for me, I am not certain if included in the installation was SQlite or Maria MySQL Database software.  Or if the automated CasaOS only creates a simple storage folder that holds the actual JPG file with no need to install any SQL database software of any kind.   

In short, I hope somebody in the community can clarify to me what storage mechanism does PhotoPrism default to when it is automatically installed using CasaOS app installer?   Once I know that, I think the mystery of where my missing test JPG file is at will be solved.  

Lastly,  I am also thinking about installing SyncThing to sync my smart phone photos to my PhotoPrism.  How will the SyncThing app be able to sync any JPG files if PhotoPrism is storing my photos in some cryptic way that I assume is going to make it difficult for the SyncThing app to recognize?   

Any advice welcome.  Thank you for your time.
